### PR TITLE
NRI Police Outfit Redesign

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Starting with the less important stuff, the old NRI police uniform is now gone, getting rid of another Flavrius sprite that didn't fit with the entirety of the codebase.
"NRI ERT" module has been renamed into, simply "NRI" module to better reflect its purpose, as it's no longer used by the NRI ERT only (Which I'll probably remove at a later date).
Now for the cool part: NRI Inspectors midround as well as the loadout have been given an entirely new outfit shamelessly copied from Disco Elysium's Revachol Citizens'/Citizen's Militia, -just like the colonial outfit-. Said outfits are, obviously, worn by the NRI police midround, among the new NRI Police armor set that's currently replacing that ancient as hell terribly green Z army Ratnik armor set, same for the NRI police gas mask.
image
image
image

## How This Contributes To The Skyrat Roleplay Experience
For the "boring" part: NRI module now actually represents what it does; unfitting sprites suck.
For the more interesting part:
Well, really though, unfitting sprites suck, since the previous NRI police uniform was overly militaristic. Ideally, this new outfit should make them feel themselves less aggressive ~~and less looking like weirdly shaded mess~~ and better represents their purpose as diplomats and inspectors instead of an actual combat personnel.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image
image
image
</details>

## Changelog

:cl: Stalkeros
add: NRI Police outfit full redesign. Sunrise, Parabellum.
imageadd: NRI Police MODsuit palette swapped.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/encounters/code/nri_raiders.dm
Comment (vinylspiders) [inline modular_skyrat/modules/encounters/code/nri_raiders.dm]:
```suggestion
	department_color = "#1f3347"
```

Could you make these colors defines like how it's done in `code/_DEFINES/colors.dm`?

You could make a new file for it e.g. `code/_DEFINES/~skyrat_defines/colors.dm`.

File: modular_skyrat/modules/encounters/code/nri_raiders.dm
Comment (vinylspiders) [inline modular_skyrat/modules/encounters/code/nri_raiders.dm]:
```suggestion
	subdepartment_color = "#c0c0c0"
```
Same thing here

File: modular_skyrat/modules/encounters/code/nri_raiders.dm
Comment (vinylspiders) [inline modular_skyrat/modules/encounters/code/nri_raiders.dm]:
```suggestion
	backpack_contents = list(
		/obj/item/storage/box/nri_survival_pack/raider = 1,
		/obj/item/ammo_box/magazine/m9mm_aps = 3,
		/obj/item/gun/ballistic/automatic/pistol/ladon/nri = 1,
		/obj/item/crucifix = 1,
		/obj/item/clothing/mask/gas/nri_police = 1,
		/obj/item/modular_computer/pda/nri_police = 1,
	)
```

**Review by vinylspiders — COMMENTED** [review]
As far as the code side of things are concerned, this looks good to me other than those nitpicks

**Review by vinylspiders — APPROVED** [review]

**Comment by vinylspiders** [comment]
Just have to wait for spritetainer approval now, code is 👍

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]:
```suggestion
/datum/loadout_item/head/imperial_police_cap
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_masks.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_masks.dm]:
```suggestion
/datum/loadout_item/mask/imperial_police_gas_mask
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm]:
```suggestion
/datum/loadout_item/neck/imperial_police_boatcloak
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm]:
```suggestion
/datum/loadout_item/shoes/imperial_police_half_boots
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm]:
```suggestion
/datum/loadout_item/suit/imperial_police_plate_carrier
```

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]:
```suggestion
/datum/loadout_item/under/jumpsuit/imperial_police_uniform
```

File: modular_skyrat/modules/new_russian_empire/code/police_outfit.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/new_russian_empire/code/police_outfit.dm]:
```suggestion
		Silver NRI police insignia is woven right above its visor."
```

File: modular_skyrat/modules/encounters/icons/pda.dmi
Comment (GoldenAlpharex) [inline modular_skyrat/modules/encounters/icons/pda.dmi:1]: Why did you have to duplicate basically all of the PDA sprites into this file? Can't you just make your PDA frame point to here and the other overlays to the base file?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Other than these few comments, this should be good to go. The sprites are nice.

File: modular_skyrat/modules/encounters/icons/pda.dmi
Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/icons/pda.dmi:1]: Answered in Discord; PDAs call for initial icon file instead of the main one for their overlays.
I'll probably change it upstream at a later date; was meant to reply to this yesterday but oh well.

**Comment by Stalkeros2** [comment]
Screw this, not feeling like it at all.

**Pull request closed without merging**
